core/store/src/main/java/uk/gov/gchq/gaffer/store/library/HashMapGraphLibrary.java

```diff
@@ -27,6 +27,14 @@
     private static final Map<String, byte[]> SCHEMAS = new HashMap<>();
     private static final Map<String, StoreProperties> PROPERTIES = new HashMap<>();
 
+    public HashMapGraphLibrary() {
+
+    }
+
+    public void initialise(final String file) {
+        // Do nothing, only used with FileGraphLibrary
```

Comment:
Don't reference the FileGraphLibrary here as there could be other libraries in the future. Change the comment to:
// Do nothing.